src/style.css

```diff
@@ -0,0 +1,20 @@
+div {
+  height: 100px;
+  width: auto;
```

Comment:
Block elements has 100% of the width of the parent container. Set width auto here is redundant